### ISSUE TITLE
Data used for Waymo

### ISSUE THREAD

**Issue by charmeleonz** [SPengLiang/LPCG#8]
Hi author, in the paper when you say 10% labelled + 90% unlabelled for Waymo experiments, do you mean 10% of the full labelled training set as labelled data + the rest 90% of the full labelled training set as unlabelled data?

**Comment by SPengLiang** [comment]
Yes, you're right, and we generate pseudo labels on the unlabeled data. Note that the choice of labeled data should be operated on the sequence level because randomly choosing labeled data can lead to data leakage due to the similarity between adjacent frames.

**Comment by charmeleonz** [comment]
@SPengLiang Thanks!
